lib/pbench/server/api/resources/users_api.py

```diff
@@ -269,14 +268,24 @@ def post(self):
                         "message": "failure message"
                     }
         """
-        auth_token = self.auth.token_auth.get_auth()["token"]
-        user = self.auth.token_auth.current_user()
+        auth_token = self.auth.get_auth_token(self.logger)
+        user = Auth.verify_auth(auth_token=auth_token)
+
+        # "None" user represents that either the token is not present in our database or it is an expired token.
+        # Expired token is already deleted by now if we reach here.
+        # In either case we will return suceess to a client, since the same token can not be used again.
 
         try:
             ActiveTokens.delete(auth_token)
-            self.logger.info(
-                "Auth token entry deleted for user with username {}", user.email
-            )
+            # Do we need to log every time someone logs out
```

Comment:
We'll want an audit log system to track basically everything; if we had one, I suppose maybe it'd be better not to waste "debug log" space with stuff like this. For now, this is our only way to track it, and I'd say the entry is worthwhile.